README.md

```diff
@@ -0,0 +1,72 @@
+# Visualization API
+
+This repository contains app that fetch (serverless) **BIRAFFE2** dataset and then visualizate it.
+
+The **"BIRAFFE2: The 2nd Study in Bio-Reactions and Faces for Emotion-based Personalization for AI
+Systems"** dataset is part of the work carried out by the AfCAI Research Group in Kraków, Poland.
+
+**BIRAFFE2** source: https://zenodo.org/record/3865860#.XvjpwecwhPY
```

Comment:
BIRAFFE is one of potentials datasets we could use. In theory we can use this to visualize anything. Put this as a subsection under #Datasets sections.


ie

```
- [Table of Contents](#table-of-contents)
- [Getting started](#getting-started)
- [Datasets](#datasets)
    -[BIRAFFE2](#biraffe2)
```